src/main/resources/static/api.yml

```diff
@@ -443,24 +443,34 @@ paths:
           required: true
           schema:
             type: string
-            format: string
+        - name: page
+          in: query
+          description: Page number of the requested page
+          required: false
+          schema:
+            type: integer
+            default: 0
```

Comment:
Just to make sure I've understood this correctly, if we don't pass any page/size params, we'll get page `0` and `10` results rather than all of them?

Reply:
That's right. It means we can implement UI changes to allow for more/all results to a page, if we wanted to.